framework/include/wifi_manager/wifi_manager.h

```diff
@@ -386,7 +386,7 @@ wifi_manager_result_e wifi_manager_disconnect_ap(void);
  *     if both SSID and channel are not set in config then it'll return error.
  *     wifi_manager check whether SSID is set by ssid_length in config. In the same way
  *     not checking channel is 0, but checking channel is greater than 0 and less than 15 in 2.5GHz.
```

Comment:
And we do not check that `channel is greater than 0` anymore.

Reply:
wifi_manager checks it with following condition.
if (config->ssid_length == 0 && config->channel == 0) {